packages/fast-components-styles-msft/src/index.ts

```diff
@@ -23,6 +23,12 @@ export { HeadingStyles };
 import HypertextStyles from "./hypertext";
 export { HypertextStyles };
 
+import ContextMenuStyles from "./context-menu";

```

Comment:
do we want to alphabetize these?

Reply:
We're not alphabetizing import sources, so not really.

Reply:
nm this is in the primary export - we do want to alphabetize here.